recipe/build.sh

```diff
@@ -1,12 +1,164 @@
 #!/bin/bash
 
-mkdir -p $PREFIX/bin
-mkdir -p $PREFIX/include
-mkdir -p $PREFIX/lib
-mkdir -p $PREFIX/share/man/man1/
+# First build patched qt4 static lib
+cd wkhtmltopdf/qt
 
+# We're linking openssl statically with the -openssl-linked argument
+# to qt's configure. So we need libopenssl-static. We also need the
+# headers from openssl. But if we add those packages to requirements
+# in meta.yml as normal, there will be conflicts and we can't do the
+# following hack to remove the .so files to force it to be linked
+# statically.
```

Comment:
This seems like an issue with the openssl feedstock, can you please open and resolve there? Or just find a way to link against the shared libraries.

Reply:
I really didn't understand what was going wrong when linking with the shared libraries in `openssl` - the build would succeed but then it failed setting up the test phase with some conflicts. When I commented out the test from `meta.yml` I was able to install the built package in a fresh conda environment and run the binary.
I wouldn't know where to start with opening an issue on `openssl` since I really don't understand what is going wrong. Would it be helpful if I update `meta.yml` on this branch to the failing version?

Reply:
Ahh yeah, that test should run though.  That means something is wrong if the test fails.

Reply:
OK I guess I must have done something wrong before. Removed this silliness from build.sh, added openssl to requirements and it seems to work. We'll see if CI passes. Thanks for your help and the speedy feedback!